src/modeler/AutoRest.Swagger.Tests/Resource/Swagger/Validation/swagger-summary-description.json

```diff
@@ -0,0 +1,177 @@
+{
```

Comment:
Concerned about more test cases?? :-) I believe every scenario must have its own test case to avoid confusions later